dpytools/config/properties/string.py

```diff
@@ -23,17 +23,20 @@ def secondary_validation_passed(self):
         Non type based validation you might want to
         run against a configuration value of this kind.
         """
+        
         if len(self.value) == 0:
             raise ValueError(f"Str value for {self.name} is an empty string")
         
         if self.regex:
             # TODO - confirm the value matches the regex
-            ...
+            pass
```

Comment:
Can you add a regex check here please. Theres some resources here:
- https://www.developer.com/languages/regular-expressions-primer/
- https://www.w3schools.com/python/python_regex.asp.

Very useful thing to get your head into, we've a lot of. regex stuff coming up.

Reply:
My bad, I told myself I would do this at the end and ended up forgetting it completely when I got caught up in the config from_env business.